x-pack/test/functional/apps/rollup_job/tsvb.js

```diff
@@ -35,7 +36,13 @@ export default function ({ getService, getPageObjects }) {
 
     before(async () => {
       // load visualize to have an index pattern ready, otherwise visualize will redirect
-      await esArchiver.load('x-pack/test/functional/es_archives/visualize/default');
+      await security.testUser.setRoles(['global_visualize_all', 'test_rollup_reader']);
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/rollup/rollup.json'
+      );
+      await kibanaServer.uiSettings.replace({
+        defaultIndex: 'rollup',
+      });
```

Comment:
Just out of curiosity, is this actually needed? It looks like you load a config object from `rollup.json` that already sets the default index to "rollup".

Reply:
@jportner I just had a discussion with @wayneseymour about this for another test he's working on.  I don't really understand why, but the "config" object we load from an archive never seems to be used.  After this kbnArchive is loaded, if you look at the saved objects, you'll find another one for the current version and it won't have the defaultIndex set.  I don't know if it's only because the version doesn't match, or if Kibana already created another one?  It could be because we don't treat the previous version config object like an upgrade case on dev builds?

Reply:
Ah, interesting! Not sure why but it's good to know for future reference!